sdk/eventgrid/Azure.Messaging.EventGrid/tests/Samples/Sample1_SendEventsToTopicAndDomain.cs

```diff
@@ -90,37 +90,43 @@ public async Task SendCloudEventsToTopic()
             string topicEndpoint = TestEnvironment.CloudEventTopicHost;
             string topicAccessKey = TestEnvironment.CloudEventTopicKey;
 
-            // Example of a custom ObjectSerializer used to serialize the event payload to JSON
-            var myCustomDataSerializer = new JsonObjectSerializer(
-                new JsonSerializerOptions()
-                {
-                    PropertyNamingPolicy = JsonNamingPolicy.CamelCase
-                });
-
             // Create the publisher client using an AzureKeyCredential
             // Custom topic should be configured to accept events of the CloudEvents 1.0 schema
             #region Snippet:CreateClientWithOptions
-
             EventGridPublisherClient client = new EventGridPublisherClient(
                 new Uri(topicEndpoint),
                 new AzureKeyCredential(topicAccessKey));
             #endregion
 
             #region Snippet:SendCloudEventsToTopic
+            // Example of a custom ObjectSerializer used to serialize the event payload to JSON
+            var myCustomDataSerializer = new JsonObjectSerializer(
+                new JsonSerializerOptions()
+                {
+                    PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+                });
+
             // Add CloudEvents to a list to publish to the topic
             List<CloudEvent> eventsList = new List<CloudEvent>
             {
-                // CloudEvent with populated data
+                // CloudEvent with custom model serialized to JSON
                 new CloudEvent(
                     "/cloudevents/example/source",
                     "Example.EventType",
-                    myCustomDataSerializer.Serialize("This is the event data")),
+                    new CustomModel() { A = 5, B = true }),
+
+                // CloudEvent with custom model serialized to JSON using a custom serializer
+                new CloudEvent(
+                    "/cloudevents/example/source",
+                    "Example.EventType",
+                    myCustomDataSerializer.Serialize(new CustomModel() { A = 5, B = true }),
+                    "application/json"),
 
                 // CloudEvents also supports sending binary-valued data
                 new CloudEvent(
                     "/cloudevents/example/binarydata",
                     "Example.EventType",
-                    new BinaryData(Encoding.UTF8.GetBytes("This is binary data")),
+                    new BinaryData(Encoding.UTF8.GetBytes("This is treated as binary data")),
                     "example/binary")};
```

Comment:
did you want to also update this to the octet-stream used elsewhere for the content-type of binary data?

Reply:
yes..

Reply:
disabling auto-merge 😄

Reply:
I'm actually not sure how I managed that considering both md files were using the same snippet.